src/store/members.store.js

```diff
@@ -0,0 +1,88 @@
+import { DisplayMemberViewModel } from '@/view_models/AccessGroupViewModel.js';
+import AccessGroups from '@/assets/js/access-groups';
+
+const accessGroupsService = new AccessGroups();
+export const membersState = {
+  members: null,
+};
+export const membersActions = {
+  async fetchMembers({ commit }, groupName) {
+    try {
+      // Change this later
+      const { members } = await accessGroupsService.getAllMembers(groupName);
+      commit('setMembers', members);
+      return members;
+    } catch (e) {
+      throw new Error(e.message);
+    }
+  },
+  async removeMember({ state, dispatch }, member) {
+    const groupName = state.group.name;
+    try {
+      const result = await accessGroupsService.deleteMember(
+        groupName,
+        member.uuid
+      );
+      return await dispatch('fetchMembers', groupName);
+    } catch (e) {
+      throw new Error(e.message);
+    }
+  },
+  async renewMember({ state, dispatch }, { memberUuid, expiration }) {
+    try {
+      const result = await accessGroupsService.renewMember(
+        state.group.name,
+        memberUuid,
+        expiration
+      );
+      return await dispatch('fetchMembers', state.group.name);
+    } catch (e) {
+      throw new Error(e.message);
+    }
+  },
+  async addCurators({ state, dispatch }, curators) {
+    try {
+      const result = await accessGroupsService.addCurators(
+        state.group.name,
+        curators
+      );
+      return await dispatch('fetchMembers', state.group.name);
+    } catch (e) {
+      throw new Error(e.message);
+    }
+  },
+  async removeCurators({ state, dispatch }, { curators, expiration }) {
+    try {
+      const result = await accessGroupsService.removeCurators(
+        state.group.name,
+        curators,
+        expiration
+      );
+      return await dispatch('fetchMembers', state.group.name);
+    } catch (e) {
+      throw new Error(e.message);
+    }
+  },
+};
+export const membersMutations = {
+  setMembers(state, memberData) {
+    try {
+      state.members = [];
+      for (const aMember of memberData) {
+        state.members.push(new DisplayMemberViewModel(aMember));
+      }
+    } catch (e) {
+      state.error = e.message;
+      throw new Error(e.message);
+    }
+  },
+};
+export const membersGetters = {
+  getMembers: ({ members }) => members,
+  getCurators: ({ members }) => members.filter(member => member.isCurator()),
+  isCurator: ({ members }) => uuid =>
```

Comment:
`isCurator` and `isMember` should be derived from the [group details endpoint](https://app.swaggerhub.com/apis/fiji/dinoparkgroups/1.0.0-oas3#/default/get_groups_api_v1_groups__groupName__details).

Reply:
Those getters determine if a specific member is a curator or a member. It's not about whether the current user is a member or curator.